docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  db:
+    image: postgres:14.15-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build: .
```

Comment:
Ensure that the Dockerfile is correctly set up in the root directory as the `build` context is set to `.`. This will build the Docker image using the Dockerfile in the current directory.